Dockerfile.QA

```diff
@@ -330,3 +330,4 @@ RUN rm -fr qa/L0_copyrights qa/L0_build_variants && \
     xargs pip3 install --upgrade
 
 ENV LD_LIBRARY_PATH /opt/tritonserver/qa/clients:${LD_LIBRARY_PATH}
+ENV LD_LIBRARY_PATH /opt/hpcx/ompi/lib:${LD_LIBRARY_PATH}
```

Comment:
Add a comment with the DLIS ticket id over this line for why it is needed.

Reply:
Added comment 👍